### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by citizen428**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [X] Documentation Update

## Description

It was brought to my attention (thanks @sarthology) that the banner in the README still listed our Ruby version as 2.7.1, so I updated that. While I was there I also fixed some other things.

## Added tests?

- [ ] No, and this is why:  documentation update

## Added to documentation?

- [ ] [Developer Docs](https://docs.forem.com) and/or [Admin Guide](https://forem.gitbook.io/forem-admin-guide/)
- [X] README
- [ ] No documentation needed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,37 +6,34 @@
 <br>
 <p align="center">
   <a href="https://www.ruby-lang.org/en/">
-    <img src="https://img.shields.io/badge/Ruby-v2.7.1-green.svg" alt="ruby version">
+    <img src="https://img.shields.io/badge/Ruby-v2.7.2-green.svg" alt="ruby version">
```
Comment (citizen428) [inline README.md:9]: This is how it started.

File: README.md [hunk 2]
```diff
@@ -6,37 +6,34 @@
...
   </a>
   <a href="http://rubyonrails.org/">
     <img src="https://img.shields.io/badge/Rails-v6.0.3-brightgreen.svg" alt="rails version">
   </a>
-  <a href="https://travis-ci.com/thepracticaldev/dev.to">
-    <img src="https://travis-ci.com/thepracticaldev/dev.to.svg?branch=master" alt="Travis Status for thepracticaldev/dev.to">
+  <a href="https://travis-ci.com/forem/forem">
```
Comment (citizen428) [inline README.md:14]: I updated remaining instances of `thepracticaldev/dev.to` to `forem/forem`.

File: README.md [hunk 3]
```diff
@@ -6,37 +6,34 @@
...
+    <img src="https://travis-ci.com/forem/forem.svg?branch=master" alt="Travis Status for forem/forem">
   </a>
-  <a href="https://codeclimate.com/github/thepracticaldev/dev.to/maintainability">
+  <a href="https://codeclimate.com/github/forem/forem/maintainability">
     <img src="https://api.codeclimate.com/v1/badges/ce45bf63293073364bcb/maintainability" alt="Code Climate maintainability">
   </a>
-  <a href="https://codeclimate.com/github/thepracticaldev/dev.to/test_coverage">
```
Comment (citizen428) [inline README.md:20]: I completely removed this, since CC said this isn't even configured for our repo.

File: README.md [hunk 4]
```diff
@@ -6,37 +6,34 @@
...
-    <img src="https://www.codetriage.com/thepracticaldev/dev.to/badges/users.svg" alt="CodeTriage badge">
+  <a href="https://www.codetriage.com/forem/forem">
+    <img src="https://www.codetriage.com/forem/forem/badges/users.svg" alt="CodeTriage badge">
   </a>
   <img src="https://badgen.net/dependabot/forem/forem?icon=dependabot" alt="Dependabot Badge">
   <a href="https://gitpod.io/from-referrer/">
     <img src="https://img.shields.io/badge/setup-automated-blue?logo=gitpod" alt="GitPod badge">
   </a>
-  <a href="https://app.netlify.com/sites/devto/deploys">
+  <a href="https://app.netlify.com/sites/docsdevto/deploys">
```
Comment (citizen428) [inline README.md:30]: I figured out the name from the PR check. Can someone verify the UUID on the next line, maybe @Zhao-Andy?

**Review by Ridhwana — COMMENTED** [review]
Hi @citizen428 - I'm seeing a broken image with Code Triage and commit activity:

Screenshot 2020-11-19 at 08 17 59

**Comment by citizen428** [comment]
Thanks @Ridhwana, I submitted forem/forem on CodeTriage now, so that should work now. Will look into the other one.

**Comment by citizen428** [comment]
BTW @Ridhwana, how are you previewing this? It still includes the shield for test coverage which I removed in this PR.

**Comment by Ridhwana** [comment]
> BTW @Ridhwana, how are you previewing this? It still includes the shield for test coverage which I removed in this PR.

Just clicked on display the rich diff.

Screenshot 2020-11-19 at 08 40 45

**Comment by citizen428** [comment]
Ha, I did not notice that there was a slight red background, thanks! 😄

**Review by Ridhwana — APPROVED** [review]
This looks good 🚀

**Review by rhymes — APPROVED** [review]
Nice catch!

**Review by benhalpern — APPROVED** [review]

**Pull request merged**